internal/storage/storage.go

```diff
@@ -353,3 +404,24 @@ func (db *Database) SearchID(ctx context.Context, url string) (int, error) {
 	return id, nil
 
 }
+
+func (db *Database) DeleteForUser(ctx context.Context, inputCh chan middleware.ItemDelete) {
+
+	for {
+		select {
+		case <-ctx.Done():
+			log.Println("case ctx.Done()")
+			return
+
+		case item := <-inputCh:
+			sql := fmt.Sprintf("UPDATE public.storage SET actual=false WHERE user_id ='%s' and id in %s;",
+				item.User, item.StringIDs)
```

Comment:
```suggestion
			sql := "UPDATE public.storage SET actual=false WHERE user_id = $1 and id =ANY($2)"
```
В виде прямой подстановки возникает риск SQL injection. 

Ссылка для самостоятельного изучения проблемы:
https://habr.com/ru/post/148151/

Reply:
изучила, спасибо